largestPrimeFactor.js

```diff
@@ -1,6 +1,21 @@
-exports.largestPrimeFactor = function(n){
-  var primeNumber = 0;
-  // do your work here
+exports.largestPrimeFactor = function(n) {
+  const primeFactors = [];
+  debugger;
```

Comment:
You shouldn't leave a `debugger` statement in your committed code. It will force a breakpoint for anyone who runs it.